contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,505 @@
+
+You only need to call `identify` once per session.
+
+### 2. Use unique strings for Distinct IDs
+
+If two users have the same distinct ID, their data will be merged and they will be considered as one user in PostHog. Two common ways in which this can happen are:
+
+- Your logic for generating IDs does not generate sufficiently strong IDs and you can end up with a clash where 2 users have the same ID.
+- There's a bug, typo, or mistake in your code leading to most or all users being identified with generic IDs like `null`, `true`, or `distinctId`.
+
+PostHog also has a few built-in protections to stop the most common distinct ID mistakes:
+
+- We do not allow identifying users with empty space strings of any length – e.g.,`' '`, `'       '`, etc.
+- We do not allow identifying users with the following strings (case insensitive):
+    - `anonymous`
+    - `guest`
+    - `distinctid`
+    - `distinct_id`
+    - `id`
+    - `not_authenticated`
+    - `email`
+    - `undefined`
+    - `true`
+    - `false`
+
+- We do not allow identifying users with the following strings (case sensitive):
+  - `[object Object]` 
+  - `NaN`
+  - `None`
+  - `none`
+  - `null`
+  - `0`
+
+- We do not allow identifying a user that has already been identified with a different distinct ID. For example:
+
+<MultiLanguage selector="tabs">
+
+```javascript
+posthog.identify(
+    'distinct_id_one',
+    {} 
+    {}, 
+);
+posthog.identify(
+    'distinct_id_two',
+    {} 
+    {}, 
+);
+// ❌ Not possible, since we already identified this user with "distinct_id_one"
+// so we cannot identify them again with a different distinct ID "distinct_id_two"
+```
+
+```Android
+PostHog.with(this)
+       .identify(distinct_id_one, new Properties());
+PostHog.with(this)
+       .identify(distinct_id_two, new Properties());
+// ❌ Not possible, since we already identified this user with "distinct_id_one"
+// so we cannot identify them again with a different distinct ID "distinct_id_two"
+```
+
+```iOS
+posthog.identify("distinct_id_one", 
+          properties: [])
+posthog.identify("distinct_id_two", 
+          properties: [])
+// ❌ Not possible, since we already identified this user with "distinct_id_one"
+// so we cannot identify them again with a different distinct ID "distinct_id_two"
+```
+
+</MultiLanguage>
+
+### 3. Reset after logout
+
+If a user logs out on your frontend, you should call `reset` to unlink any future events made on that device with that user. 
+
+This is important if your users are sharing a computer, as otherwise all of those users will be grouped together into a single user due to shared cookies between sessions. **We strongly recommend you do this on logout even if you don't expect users to share a computer.**
```

Comment:
```suggestion
This is important if your users are sharing a computer, as otherwise all of those users are grouped together into a single user due to shared cookies between sessions. **We strongly recommend you call `reset` on logout even if you don't expect users to share a computer.**
```